src/store/entity/getters.ts

```diff
@@ -26,4 +26,24 @@ export const getters: GetterTree<Entity, any> = {
 	getAliasesByLanguage: ( state: Entity ) => ( languageCode: string ): Term[] | null => {
 		return state.aliases[ languageCode ] || null;
 	},
+	getAvailableLanguageKeys: ( state: Entity ): string[] => {
```

Comment:
I think this is a great use case for a getter!

Naming suggestion
`getAllEnteredLanguages`/`getAllEnteredLanguageCodes`?